config/locales/de.yml

```diff
@@ -332,37 +332,30 @@ de:
         ausgewählt </span>"
       message: Sie haben %{cost} für %{project} ausgewählt
     adjustable_token_project: ##fixme language
-      instructions: "Please select one of the options below:"
-      not_adjusted_warning: "<span style='color:red'>You have selected 0 tokens for %{project}</span>"
-      message: "You have selected %{cost} tokens for %{project}"
+      instructions: "Bitte wählen Sie eine der folgenden Optionen:"
+      not_adjusted_warning: "<span style='color:red'>Sie haben 0 Punkte für %{project} ausgewählt.</span>"
+      message: "Sie haben %{cost} Punkte für %{project} ausgewählt."
 
   token:
     title: "Participatory Budgeting"
     popup:
-      title: "Instructions"
-      body: "<p style='font-size: 20px;'>You will now be asked to participate in an activity that
-will <u>not</u> affect your vote, but is part of a Stanford University research project meant
-to help us improve the participatory budgeting process.</p>
-<ul style='font-size: 20px;'><li>Participating in this activity is optional. You can exit at any time.</li>
-<li>The expected time for this activity is 2-3 minutes.</li>
+      title: "Anleitung"
+      body: "<p style='font-size: 20px;'>Sie werden nun aufgefordert, an einer Aktivität teilzunehmen, die Ihre Abstimmung <u>nicht</u> beeinflusst,
+sondern Teil eines Forschungsprojekts der Stanford University ist, das uns helfen soll, den partizipatorischen Budgetprozess zu verbessern.</p>
+<ul style='font-size: 20px;'><li>Die Teilnahme an dieser Aktivität ist optional. Sie können sie jederzeit beenden.</li>
+<li>Die erwartete Dauer für diese Aktivität beträgt 2-3 Minuten.</li>
```

Comment:
Would you be able to make this a translation of the original text? Just making sure English and German are in sync. When you're running locally, you can of course overrule this with your own texts.